transfers/seed.py

```diff
@@ -1,29 +1,204 @@
-from db.thing import Thing
+"""
+Populates the database with interconnected fake data for frontend CI testing.
+
+Run with:
+    docker compose exec -T app python -m transfers.seed
+"""
+
+import random
+from datetime import datetime, timedelta
+from faker import Faker
 from db.engine import session_ctx
 
+# Core models
+from db.contact import Contact
+from db.location import Location
+from db.thing import Thing
+from db.sensor import Sensor
+from db.deployment import Deployment
+from db.sample import Sample
+from db.observation import Observation
+from db.parameter import Parameter
+from db.analysis_method import AnalysisMethod
+from db.regulatory_limit import RegulatoryLimit
+from db.transducer import TransducerObservation
+from db.status_history import StatusHistory
+
+fake = Faker()
+Faker.seed(42)
+random.seed(42)
+
+
+def seed_all(n=5):
+    """Seed roughly `n` of each main entity and connect them."""
+    with session_ctx() as s:
+        contacts = []
+        locations = []
+        things = []
+        sensors = []
+        parameters = []
+        methods = []
+        samples = []
+        observations = []
+
+        # 1. Contacts
+        for _ in range(n):
+            c = Contact(
+                name=fake.name(),
+                organization=fake.company(),
+                role=random.choice(["Hydrologist", "Technician", "Geologist"]),
+                contact_type="Person",
+            )
+            s.add(c)
+            contacts.append(c)
+
+        # 2. Locations
+        for _ in range(n):
+            loc = Location(
+                name=fake.city(),
+                latitude=round(fake.latitude(), 6),
+                longitude=round(fake.longitude(), 6),
+                release_status="public",
+            )
+            s.add(loc)
+            locations.append(loc)
+
+        # 3. Parameters & Methods
+        param_names = ["pH", "Temperature", "Conductivity", "TDS", "DO"]
+        for p in param_names:
+            prm = Parameter(
+                parameter_name=p,
+                matrix="Water",
+                default_unit="pH units" if p == "pH" else "mg/L",
+            )
+            s.add(prm)
+            parameters.append(prm)
```

Comment:
This isn't needed if `MODE=development` since setting that environment variable initializes both the parameter and lexicon tables. See `core/appy.py`